bbot/modules/baddns.py

```diff
@@ -26,10 +26,17 @@ class baddns(BaseModule):
     max_event_handlers = 8
     deps_pip = ["baddns~=1.1.789"]
 
+    # We allow distance-1 events in at the module level so we can selectively block them in filter_event
+    scope_distance_modifier = 1
+
+    def _incoming_dedup_hash(self, event):
+        return hash(f"{event.host}|{event.type}")
```

Comment:
I think `per_host_only` was the correct solution here. You're handling all events the same regardless of event type, so it seems like this will cause the same host to be processed multiple times.

Reply:
No, this is definitely necessary because we are dealing with them differently in the filter_event.

if a DNS_NAME comes in first, it will block the URL that has the extension-js tag that we want to let through if we don't de-dedupe by event type.